practice/practice-02/src/index.pug

```diff
@@ -1,8 +1,25 @@
+include nav-item
+
 doctype html
 html(lang="en")
   head
     meta(charset="UTF-8")
     meta(name="viewport", content="width=device-width, initial-scale=1.0")
     link(rel="stylesheet", href="./sass/main.scss")
     title Food Hut
-  body 
\ No newline at end of file
+  body
+    header.header
+      .logo-wrapper 
+        a(href="javascript:void(0)") 
+          img(src="./assets/images/Logo.png", alt="Logo image").logo
```

Comment:
Pls fix:
1. File name of image
2. Alternative content

Reply:
Fixed.